### ISSUE TITLE
Bug: Search provider finds reults, but does not open them

### ISSUE THREAD

**Issue by dschier-wtd** [flathub/org.gnome.Contacts#12]
When using the contacts search provider, one will find names. Opening these names results in a "contact not found".

![Screenshot from 2020-05-24 08-49-27](https://user-images.githubusercontent.com/6490538/82747653-9267f680-9d9b-11ea-9d70-65a03168abfa.png)

OS: Fedora 32

```
~ $ flatpak info org.gnome.Contacts

Contacts - A contacts manager for GNOME

          ID: org.gnome.Contacts
         Ref: app/org.gnome.Contacts/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 3.36
     License: GPL-2.0-or-later
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 26,3 MB
     Runtime: org.gnome.Platform/x86_64/3.36
         Sdk: org.gnome.Sdk/x86_64/3.36

      Commit: a2e8974d4d666c9a8b8a98f683d01db67a4ed32ffa5f0877fe3ef3ce45752b66
      Parent: 1ca41b6c454cbe0a77e3c593bc931d9349746d98a99b7a609d93a11448d8a31e
     Subject: Update gnome-desktop, and housekeeping (dd2ddcda)
        Date: 2020-04-01 17:40:53 +0000
```

**Comment by nielsdg** [comment]
Upstream issue: https://gitlab.gnome.org/GNOME/gnome-contacts/-/issues/167

Problem should be solved in 3.38.1, which should be updated in Flathub soon (#14)

**Comment by daniel-schier** [comment]
works lika charm. Thank you :)
